funding/manager.go

```diff
@@ -1021,7 +1021,14 @@ func (f *Manager) stateStep(channel *channeldb.OpenChannel,
 
 		if !received {
 			// We haven't received FundingLocked, so we'll continue
-			// to the next iteration of the loop.
+			// to the next iteration of the loop after sleeping for
+			// one second.
+			select {
+			case <-time.After(1 * time.Second):
```

Comment:
In `receiveFundingLocked`, there is also a comment

`// Avoid a tight loop if peer is offline.`

Shouldn't the tight-loop-prevention code all be in one place?

Reply:
It's slightly different as we are not just waiting for peer here, but also waiting for an internal state as we check `channel.RemoteNextRevocation != nil`. That's why I added the log below to differentiate the two cases.

Also I'm not in favor of this style, but just need a way to stop the CPU spike. I think we'll need more work here, see [previous discussion](https://github.com/lightningnetwork/lnd/pull/6664#discussion_r919087397).